lib/database/ticket.php

```diff
@@ -114,10 +114,10 @@ function submitNewTickets($userSubmitter, $idsCSV, $reportType, $noteIn, &$summa
         return false;
     }
 
-    sanitize_sql_inputs($userSubmitter, $reportType, $noteIn);
+    $note = $noteIn;
+    sanitize_sql_inputs($userSubmitter, $reportType, $note);
```

Comment:
sanitize a copy of the message so the unescaped value can be passed to later functions.